dts/arm/nxp/nxp_lpc54xxx.dtsi

```diff
@@ -39,7 +30,10 @@
 	soc {
 
 		flash0:flash@0 {
-			reg = <0 0x40000>;
+			/* reduced from 0x40000 to 0x30000
+			 * to leave room for secondary core
+			 */
+			reg = <0 0x30000>;
```

Comment:
Would it be better to use partitions instead, like we do for mcuboot?

Reply:
The partition names currently available for mcuboot might be a little misleading for this case, but I can try to change it.